src/views/product-integration/component-view/components/variable-group-list/index.tsx

```diff
@@ -105,6 +105,7 @@ export function VariableGroupList({
 										</span>
 									)}
 									<MdxHeadingPermalink
+										ariaLabel={`${variable.key} permalink`}
```

Comment:
Adjacent required change, this should have been here before, but didn't catch it till I added types for `MdxHeadingPermalink` 👍